### PR TITLE
Update connections.rst

### PR DESCRIPTION
**Pull request by srpyprime**

### CODE REVIEW

File: doc/source/user_guide/connections.rst [hunk 1]
```diff
@@ -18,12 +18,30 @@ There are three major operations for zonelet connections:
 - The :func:`Connect.intersect_face_zonelets() <ansys.meshing.prime.Connect.intersect_face_zonelets>` method allows you
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:18]: Is this a function (as shown by the role) or a method as indicated in the text. They should both be using the correct role/term.

File: doc/source/user_guide/connections.rst [hunk 2]
```diff
@@ -18,12 +18,30 @@ There are three major operations for zonelet connections:
...
 
 - The :func:`Connect.join_face_zonelets() <ansys.meshing.prime.Connect.join_face_zonelets>` method allows you to join
   a set of face zonelets to another set of face zonelets along the overlapping faces. 
 
+.. figure:: ../images/connect_join.png
+    :width: 200pt
+    :align: center
+
+-	The :func:`Connect.fuse_face_zonelets() <ansys.meshing.prime.Connect.fuse_face_zonelets>` method allows you to 
+  perform fuse between overlapping face zonelets within a single part.
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:40]: "to perform fuse between" seems like awkward wording. Can this be rewritten?

File: doc/source/user_guide/connections.rst [hunk 3]
```diff
@@ -138,6 +156,48 @@ The following example shows how to accomplish these tasks:
        n_multi_edges :  9
        n_duplicate_faces :  0
 
+To perform Fuse operation,
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:159]:
```suggestion
To perform the fuse operation:

```

File: doc/source/user_guide/connections.rst [hunk 4]
```diff
@@ -138,6 +156,48 @@ The following example shows how to accomplish these tasks:
        n_multi_edges :  9
        n_duplicate_faces :  0
 
+To perform Fuse operation,
+
+1. Connect face zonelets in the model using fuse operation. You can apply Fuse on overlapping faces
+   within  a single part. You can colocate, merge, or remove fused surfaces as per your requirement.
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:162]:
```suggestion
1. Connect face zonelets in the model using the fuse operation. You can apply a fuse on overlapping faces
   within a single part. You can co-locate, merge, or remove fused surfaces as per your requirement.

```

File: doc/source/user_guide/connections.rst [hunk 5]
```diff
@@ -138,6 +156,48 @@ The following example shows how to accomplish these tasks:
...
+
+1. Connect face zonelets in the model using fuse operation. You can apply Fuse on overlapping faces
+   within  a single part. You can colocate, merge, or remove fused surfaces as per your requirement.
+
+.. figure:: ../images/fuse.png
+    :width: 200pt
+    :align: center
+
+2. Enable the fuse parameters as per your requirement. When absolute_tolerance is True, 
+   provides the gap tolerance or side tolerance value as absolute value.
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:169]:
```suggestion
2. Enable the fuse parameters as per your requirement. When ``absolute_tolerance=True``, 
   you must provide the gap tolerance or side tolerance value as an absolute value.

```

File: doc/source/user_guide/connections.rst [hunk 6]
```diff
@@ -138,6 +156,48 @@ The following example shows how to accomplish these tasks:
...
+       params.fuse_option = prime.FuseOption.TRIMONESIDE
+       params.check_interior = True
+       params.check_orientation = False
+       params.local_remesh = True
+       params.separate = True
+       params.dump_mesh = False
+       params.n_layers = 2
+       params.angle = 45
+
+fuse_option specifies how to treat the surface when performing fuse operation. Here, TRIMONESIDE option deletes the faces to be fused on one side and merges the nodes in the middle location. When local_remesh is True, meshes the fused region after performing fuse operation. n_layers specify the number of layers around the region to be fused. When separate is True, separates the fused region.
```
Comment (PipKat) [inline doc/source/user_guide/connections.rst:186]:
```suggestion
The ``fuse_option`` parameter specifies how to treat the surface when performing the fuse operation. Here, the ``TRIMONESIDE`` value results in deleting the faces to be fused on one side and merging the nodes in the middle location. When ``local_remesh = True``, the fused region is meshed after performing the fuse operation. The ``n_layers`` parameter specifies the number of layers around the region to be fused. When ``separate = True``, the fused region is separated.

```

File: doc/source/user_guide/connections.rst (hunk 3 above)
Comment (srpyprime) [reply doc/source/user_guide/connections.rst:159]: @PipKat It is the name of the operation.

**Review by waltersma — APPROVED** [review]

**Pull request merged**